File: Prianly/app/src/main/java/com/example/prianly/MainActivity.java
package com.example.prianly;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import android.os.Bundle;
import android.view.MenuItem;

import com.google.android.material.bottomnavigation.BottomNavigationView;
import com.google.android.material.navigation.NavigationBarView;

public class MainActivity extends AppCompatActivity {

    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //move `mOnNavigationItemSelectedListener` code here .
        BottomNavigationView.OnNavigationItemSelectedListener mOnNavigationItemSelectedListener
                = new BottomNavigationView.OnNavigationItemSelectedListener() {

            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                Fragment selectedFragment = null;

                switch (item.getItemId()) {
                    case R.id.navigation_home:
                        selectedFragment = new HomeFragment();
                        break;
                    case R.id.navigation_list_lyrics:
                        selectedFragment = new ListLyricsFragment();
                        break;
                    case R.id.navigation_account:
                        selectedFragment = new AccountFragment();
                        break;
                }

                return true;
            }
        };

//        BottomNavigationView navigation = (BottomNavigationView) findViewById(R.id.nav_view);
//        navigation.setOnNavigationItemSelectedListener(mOnNavigationItemSelectedListener);


    }
}

File: Prianlyy/app/src/main/java/com/example/prianlyy/MainActivity.kt
package com.example.prianlyy

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ImageView
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        ImageView leftIcon = findViewById(R.id.left_icon);
        ImageView rightIcon = findViewById(R.id.right_icon);
        TextView tittle = findViewById(R.id.toolbar_tittle);

        leftIcon.setOnClickListener(new View.OnClickListener(){
            @override
            public void onClick(view v){
                Toast.makeText(MainActivity.this, "You clicked in left icon", Toast.LENGTH_SHORT).show();
            }
        });
        rightIcon.setOnClickListener(new View.OnClickListener(){
            @override
            public void onClick(view v){
                Toast.makeText(MainActivity.this, "You clicked in right icon", Toast.LENGTH_SHORT).show();
            }
        });

        tittle.setText("Prianly");
    }
}